src/Nri/Ui/PremiumCheckbox/V8.elm

```diff
@@ -269,7 +283,7 @@ viewLockedButton { idValue, label, containerCss, onLockedMsg } =
             Nothing ->
                 Extra.none
         ]
-        [ viewPremiumFlag
+        [ viewPremiumFlag contentPremiumFlag
         , Html.span
             [ css
                 [ outline Css.none
```

Comment:
if we're cool with the padding around this then I'm cool. Right now as you change the `premiumDisplay` dropdown, "Premium Locked" moves the whole thing up because all of the checkboxes have `padding: 5px 0` while this has `padding: 0` on 293. The shift disappears when I do `padding2 (px 5) zero` there. This isn't introduced here but it's maybe worth changing? But if that's a load bearing thing in the UI as is then forget about it.

So I'm an approval if we know that this is OK, but not merging without design approval.


https://github.com/NoRedInk/noredink-ui/assets/75996561/e2cad0bb-bc78-425e-8cff-59aa123024ac

Reply:
I'm reluctant to make any general changes to styling that might currently be compensated for in-app, which might automatically propagate and change appearance in places we don't mean it to.

If it's problematic, I'm happy to add it to the backlog for after premium vouchers when there's the time to safely investigate and modify any resulting changes, but I'm not sure this is the best place/time to do so.

Reply:
ok nice, as long as it preexists our changes I'm set